### PR TITLE
[v7r2] Add histogram of CPU usage to Job accounting

### PR DESCRIPTION
**Pull request by zmathe**
It is depend on #4497 

BEGINRELEASENOTES

*AccountingSystem
NEW: Add histogram of CPU usage to Job accounting
NEW: Implement the method used to retrieve data for creating histograms
ENDRELEASENOTES

### CODE REVIEW

File: AccountingSystem/private/Plotters/BaseReporter.py [hunk 1]
```diff
@@ -295,6 +295,37 @@ def _getSummaryData(
...
+                                        endTime,
+                                        selectFields,
+                                        condDict)
+    if not retVal['OK']:
+      return retVal
+    dataDict = self._groupByField(0, retVal['Value'])
+    data = {}
+    # convert values to the correct format
+    for key, values in dataDict.items():
+      data[key] = [float(i[0]) for i in values]
```
Comment (fstagni) [inline AccountingSystem/private/Plotters/BaseReporter.py:326]: I think the last 3 lines can simply be:
```
data = {key: float(i[0]) for key, i in dataDict.items()}
```

File: AccountingSystem/private/Plotters/BaseReporter.py [hunk 2]
```diff
@@ -295,6 +295,37 @@ def _getSummaryData(
...
 
+  def _getBucketData(self, startTime, endTime, selectFields, preCondDict, groupingFields, metadataDict):
+    """
+    It retrieves data for histogram.
+
+    :param int startTime: epoch time
+    :param int endTime: epoch time
+    :param list selectFields: the value being plotted
+    :param dict preCondDict: plot attributes (conditions)
+    :param dict metadataDict: extra arguments used to create the plot.
```
Comment (fstagni) [inline AccountingSystem/private/Plotters/BaseReporter.py:306]: `groupingFields` is missing.

Comment (zmathe) [reply AccountingSystem/private/Plotters/BaseReporter.py:306]: in fact it is not needed.

**Comment by zmathe** [comment]
@fstagni This PR fails because it is depend on #4497 and it is not merged. As soon as #4497 will be merged, we can restart the tests

**Pull request merged**